src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -169,7 +178,8 @@ public static IEnumerable<object[]> InvalidNtlmNegotiateAuthentication_TestData(
             yield return new object[] { "abcd", HttpStatusCode.BadRequest };
         }
 
-        [ConditionalFact(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementationAndNotUap))] // [PlatformSpecific(TestPlatforms.Windows, "Managed impl doesn't support NTLM")]
```

Comment:
Unless the issue is closed, you should bring them back. It's there to help with searches and to match it with the affected issue: the test is effectively disabled for Unix by saying it should only run if using the Windows implementation.